frontend/src/pages/profile.tsx

```diff
@@ -8,6 +8,7 @@ import { onAuthStateChanged, getAuth } from "firebase/auth";
 import { auth } from "@/utils/firebase";
 import { useAuth } from "@/utils/AuthContext";
 import Avatar from "@/components/molecules/Avatar";
+import { Modal } from "@mui/material";
```

Comment:
Here you import `Modal` from material UI but we already have a custom built modal in `molecules/Modal`